eng/tools/TypeSpecValidation/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+      "target": "es2017",
+      "module": "commonjs",
+      "outDir": "./dist"
+    },
+    "include": ["/*"],
+    "exclude": []
```

Comment:
```suggestion
    "extends": "@tsconfig/node16/tsconfig.json"
```

Reply:
We might need to keep some of the original settings, but let's see if things work with only the base.